src/client/fontengine.cpp

```diff
@@ -287,11 +287,21 @@ gui::IGUIFont *FontEngine::initFont(const FontSpec &spec)
 			break;
 	}
 
+	bool use_freetype_outline = g_settings->getBool("use_freetype_outline");
```

Comment:
Perhaps it is better to check the existing settings and if they are not set (`== NULL`, or `== ""`), then use freetype outline?
Separately for italic, separately for bold.